cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -109,14 +109,14 @@
 
   INSTANCE_URI("cdap-instance-uri"),
   VERIFY_SSL_CERT("verify-ssl-cert"),
-  ENTITY("entity-id"),
 
   /**
    * Metadata
    */
+  ENTITY("entity-id"),
```

Comment:
``entity-id`` is not a metadata-only entity. It is used in other places too, like security